src/nxdoc/web-ui/web-ui-release-notes/web-ui-release-notes-3-0-23.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Version 3.0.23
+description: Discover what's new in Web UI 3.0.23.
+review:
+    comment: ''
+    date: '2023-05-23'
+    status: ok
+toc: true
+labels:
+tree_item_index: 981
+hidden: true
+---
+
+{{{multiexcerpt 'matching-notes' page='web-ui-release-notes'}}}
+
+{{! multiexcerpt name='web-ui-updates'}}
+## What’s New in Web UI for LTS 2021 (Version 3.0.23)
+
+This is a bugfix release.
+
+
+### Other Noteworthy Changes
```

Comment:
```suggestion
### Noteworthy Changes
```

Reply:
We shouldn't say "Other" since there's nothing else in this release :)

Reply:
Done